doc/user_manual/sampler.tex

```diff
@@ -133,6 +133,16 @@ \section{Samplers}
  \nb{Alternatively, this node must be ommitted
   if the \xmlNode{distribution} node is supplied.}
 }
+\newcommand{\shapeDescription}
+{
+  \xmlNode{shape}, \xmlDesc{comma-separated integers,
+  optional field}, determines the number of samples and shape of samples
+  to be taken.  For example, \xmlNode{shape}2,3\xmlNode{/shape} will provide
+  a 2 x 3 matrix of values.  In general, samplers will simply replicate
+  the single sampled value into each point of the matrix. Omitting this optional
+  node will result in a single scalar value instead.  Note that a model interface
+  must be prepared to handle non-scalar inputs to use this option.
```

Comment:
I feel like a system of registering and checking might be burdensome (although slightly less so if a code has to "opt in" to multivariable sampling).  Something like this could be implemented on a model-by-model basis.